packages/react/.storybook/webpack.config.js

```diff
@@ -111,7 +48,8 @@ class FeatureFlagProxyPlugin {
 }
 
 module.exports = ({ config, mode }) => {
-  config.devtool = useStyleSourceMap ? 'source-map' : '';
+  config.devtool =
+    NODE_ENV === 'production' ? 'source-map' : 'cheap-module-eval-source-map';
```

Comment:
```suggestion
    NODE_ENV === 'development' ? 'source-map' : 'cheap-module-eval-source-map';
```
are we enabling full source maps in production?

Reply:
I think that'd be ideal, what do you think?

Reply:
I haven't checked this with the flag set to production yet but if the build times are still ok then it should be fine? not sure

Reply:
I think for prod builds (`yarn build-storybook`) should be fine with timing, definitely understand for local though but I think `cheap-module-eval-sourcemap` tied with commenting out what you don't need will be good enough compared to before.